executor/join.go

```diff
@@ -530,24 +532,33 @@ func (e *NestedLoopJoinExec) prepare() error {
 	}
 }
 
-func (e *NestedLoopJoinExec) fillRowWithNullValue(row *Row) *Row {
-	newRow := &Row{
-		RowKeys: row.RowKeys,
-		Data:    make([]types.Datum, len(row.Data)+e.SmallExec.Schema().Len()),
+func (e *NestedLoopJoinExec) fillRowWithDefaultValue(bigRow *Row) (returnRow *Row) {
+	smallRow := &Row{
+		Data: make([]types.Datum, e.SmallExec.Schema().Len()),
 	}
-	copy(newRow.Data, row.Data)
-	return newRow
+	copy(smallRow.Data, e.defaultValues)
```

Comment:
No, before this pr, only nested loop join with left as outer is used in apply, and default values seem to be nil always. So it is okay before.